services/rules-faas/tests/applyRules.test.js

```diff
@@ -0,0 +1,122 @@
+const applyRules  =  require('../src/applyRules') 
+const uuidv1 = require('uuid/v1')
+const TAX_TRANSACTION_NAME = '9% state sales tax'
+
+const transaction_items = [
+    {
+      name: 'Milk',
+      price: '4.43',
+      quantity: '2',
+      author: 'Joe Smith',
+      debitor: 'Joe Smith',
+      creditor: 'Mary'
+    },
+    {
+      name: 'Bread',
+      price: '6.25',
+      quantity: '5',
+      author: 'Joe Smith',
+      debitor: 'Joe Smith',
+      creditor: 'Mary'
+    },
+    {
+      name: 'Vitamins',
+      price: '24.23',
+      quantity: '1',
+      author: 'Joe Smith',
+      debitor: 'Joe Smith',
+      creditor: 'Mary'
+    },
+    {
+      name: 'NY Steak',
+      price: '12.54',
+      quantity: '2',
+      author: 'Joe Smith',
+      debitor: 'Joe Smith',
+      creditor: 'Mary'
+    },
+    {
+      name: 'Grapes',
+      price: '5.223',
+      quantity: '1',
+      author: 'Joe Namath',
+      debitor: 'Joe Smith',
+      creditor: 'Mary'
+    },
+    {
+    name: 'Paper Towels',
+    price: '12.3348',
+    quantity: '1',
+    author: 'Joe Namath',
+    debitor: 'Joe Smith',
+    creditor: 'Mary',
+    },
+  
+    // {
+    //   name: '9% Sales Tax',
+    //   price: '7.02',
+    //   quantity: '10',
+    //   author: 'Joe Smith',
+    //   debitor: 'Joe Smith',
+    //   creditor: 'StateOfCalifornia'
+    // },
+    // {
+    //   name: '9% Sales Tax',
+    //   price: '7.02',
+    //   quantity: '10',
+    //   author: 'Joe Smith',
+    //   debitor: 'Joe Smith',
+    //   creditor: 'StateOfCalifornia'
+    // },
+    // {
+    //   name: '9% Sales Tax',
+    //   price: '7.02',
+    //   quantity: '10',
+    //   author: 'Joe Smith',
+    //   debitor: 'Joe Smith',
+    //   creditor: 'StateOfCalifornia'
+    // }
+]
+
+const with_sales_tax = applyRules( uuidv1(), transaction_items )
```

Comment:
spaces **between** arguments only.
remove space before initial argument and after last argument